app/src/main/java/com/akiniyalocts/tail/ui/cocktailDetail/CocktailDetailsDisplayModel.kt

```diff
@@ -0,0 +1,70 @@
+package com.akiniyalocts.tail.ui.cocktailDetail
+
+import com.akiniyalocts.tail.api.model.Drink
+
+data class Ingredient(
+    val name: String,
+    val quantity: String,
+) {
+    override fun toString(): String {
+        return listOf(quantity, name)
+            .filter { it.isNotBlank() }
+            .joinToString("")
+    }
+}
+
+data class CocktailDetailsDisplayModel(
+    val id: String,
+    val name: String,
+    val altName: String?,
+    val glassware: String?,
+    val ingredients: List<Ingredient>,
+    val instructions: String,
+    val image: String?,
+    val tags: List<String>,
+) {
+    companion object {
+        fun fromDrink(drink: Drink): CocktailDetailsDisplayModel {
+            val ingredientTuples = listOf(
```

Comment:
The serialization response for some of the models like `Drink` are annoying to work with. There is probably a better way to organize that upstream in the repo :thinking: